src/Symfony/Bundle/FrameworkBundle/Secrets/SodiumVault.php

```diff
@@ -107,16 +103,22 @@ public function reveal(string $name): ?string
             return null;
         }
 
+        if (!\function_exists('sodium_crypto_box_seal')) {
+            $this->lastMessage = sprintf('Secret "%s" cannot be revealed as the "sodium" PHP extension missing. Try running "composer require paragonie/sodium_compat" if you cannot enable the extension."', $name);
+
+            return null;
```

Comment:
should we return `null` silently here, or throw an exception when trying to use `reveal` without the sodium extension ? `seal` is throwing.

Reply:
we should return `null`, that's the design here - it allows e.g. the env var process to take over and provide a fallback behavior